tests/unicon/Makefile

```diff
@@ -0,0 +1,28 @@
+
+include ../Makedefs
+
+ifneq ("$(wildcard $(UC))","")
```

Comment:
Let us not mask this error. I opened a PR to fix the issue.

Reply:
To clarify, the debian package test , builds a debian  package and installs it. The unicon used should be the installed version, and that is determined by setting the UC variable in the test environment. That was correct in once place, but it looks like another instance was added at some point without taking the environment variable into account.

Reply:
I'm genuinely at a loss here. I said "The problem is in the Debian-Pkg". You said "but your test fails and the others don't" (and didn't merge the PR). So I fixed the test to not fail. Then you said "let's not mask the error" and fixed the Debian-Pkg problem (and still didn't merge this PR). I could **guess** what modifications are needed (if any) in order to merge this PR, or you could say explicitly: I very much prefer the latter.  Just to be clear, after PR367 is merged, either this version of PR363 or the previous version will both work -- which one do you want? If something else, then please say what the something else is.
As far as I can see, the other tests also use a variant of the technique I used to prevent the failure. If there's something wrong with it then there's also something wrong with the other tests (although they are testing for a feature, the skip is also selected if Unicon is absent).
I've merged PR367, rebased this commit on top of PR367 and force pushed it.

Reply:
Apologies for the confusion.  It was obvious there is an issue in the debian pkg platform, not funding the unicon binaries. But all other tests were passing. There is no explicit "skip if we don't have Unicon", so I figured we handle that case globally and that there was still a separate issue in the PR. When you added the explicit step and it worked, I decided to go fix the platform issue. 

To be clear, we don't want to have explicit skips if unicon doesn't exist,  failing  in that case is fine. If we get to the point where we don't have unicon binaries then we have a more fundamental issue in the test.